src/components/Error/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Error';
```

Comment:
Consider explicitly naming the module when exporting to avoid any assumptions about the call stack.